de/whats_next/README.md

```diff
@@ -10,33 +10,33 @@ Und dann folge Django Girls doch auf [Facebook](http://facebook.com/djangogirls)
-- [Hello Web App: Learn How to Build a Web App](https://hellowebbooks.com/learn-django/) - du kannst auch eine kostenlose eBook-Lizenz beantragen, indem du der Autorin Tracy Osborn auf <tracy@limedaring.com> schreibst
+- [Video-Lektionen "Getting Started With Django"](http://www.gettingstartedwithdjango.com/)
+- [Hello Web App: Learn How to Build a Web App](https://hellowebbooks.com/learn-django/) – du kannst auch eine kostenlose eBook-Lizenz beantragen, indem du der Autorin Tracy Osborn auf <tracy@limedaring.com> schreibst
 
 #### HTML, CSS und JavaScript
 
-- [Web Development Kurs auf Codeacademy](https://www.codecademy.com/learn/paths/web-development)
+- [Web-Development-Kurs auf Codeacademy](https://www.codecademy.com/learn/paths/web-development)
 - [freeCodeCamp](https://www.freecodecamp.org/)
 
 #### Python
 
-- [Python-Kurs auf Codeacademy](https://www.codecademy.com/learn/learn-python)
-- [Googles Python Kurs](https://developers.google.com/edu/python/)
-- [Learn Python The Hard Way](http://learnpythonthehardway.org/book/) – die ersten Übungen sind frei
+- [Python-Kurs auf Codecademy](https://www.codecademy.com/learn/learn-python)
+- [Googles Python-Kurs](https://developers.google.com/edu/python/)
+- [Learn Python The Hard Way](http://learnpythonthehardway.org/book/) – die ersten Übungen sind kostenlos
 - [New Coder Tutorials](http://newcoder.io/tutorials/) – eine Vielzahl von praktischen Beispielen, wie Python verwendet werden kann
-- [edX](https://www.edx.org/course?search_query=python) – die meisten Kurse sind kostenfrei, aber wenn du ein Zertifikat oder Credits für eine Weiterbildung erhalten willst, dann kostet das etwas
+- [edX](https://www.edx.org/course?search_query=python) – die meisten Kurse kannst du kostenfrei testen, aber wenn du ein Zertifikat oder Credits für eine Weiterbildung erhalten willst, dann kostet das etwas
 - [Courseras Python-Kurse](https://www.coursera.org/specializations/python) – einige Video-Vorträge sind kostenlos
 
 #### Mit Daten arbeiten
 
-- [Der Data Science Kurs von Codecademy](https://www.codecademy.com/learn/paths/data-science)
-- [edX](https://www.edx.org/course/?search_query=python&subject=Data%20Analysis%20%26%20Statistics) – die meisten Kurse sind kostenfrei, aber wenn du ein Zertifikat oder Credits für eine Weiterbildung erhalten willst, dann kostet das etwas
+- [Der Data-Science-Kurs von Codecademy](https://www.codecademy.com/learn/paths/data-science)
+- [edX](https://www.edx.org/course/?search_query=python&subject=Data%20Analysis%20%26%20Statistics) – die meisten Kurse kanst du kostenfrei testen, aber wenn du ein Zertifikat oder Credits für eine Weiterbildung erhalten willst, dann kostet das etwas
 - [Dataquest](https://www.dataquest.io/) – die ersten 30 "Missionen" sind frei
 
-Wir können es nicht abwarten zu sehen, was du als Nächstes baust!
\ No newline at end of file
+Wir gespannt darauf, was du als Nächstes baust!
```

Comment:
oops, missed the verb here. Will fix on Crowdin

Reply:
> Will fix on Crowdin

done

Reply:
...and Done!